src/test/java/org/assertj/core/api/iterable/IterableAssert_extracting_Test.java

```diff
@@ -65,6 +65,14 @@ public Integer extract(Employee input) {
     }
   };
 
+  private static final ThrowingExtractor<Employee, Object, Exception> throwingExtractor = new ThrowingExtractor<Employee, Object, Exception>() {
```

Comment:
use a lambda since it's for java 8

Reply:
ok

Reply:
@nach-o-man I'm currently integrating it, don't worry, thanks!